### PR TITLE
Update Dependencies Sass and Cypress

### PR DESCRIPTION
**Pull request by nicorithner**
- Switch to yarn
- Replace node-sass with sass
- Update cypress to latest version
- Update configuration using cypress ui
- Update hsla to new sass supported syntax

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor

## Concerns
-  Any bugs still present?
-  Next steps?

### CODE REVIEW

File: src/styles/i-settings/_colors.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* === Neutrals === */
-$color-light: hsla(0, 0, 100, 1);
-$color-dark: hsla(0, 0, 0, 1);
+$color-light: hsla(0, 0%, 100%, 1);
```
Comment (nicorithner) [inline src/styles/i-settings/_colors.scss]: Sass requires you have a % sign next to saturation and lightning
https://sass-lang.com/documentation/modules/color

File: package.json [hunk 2]
```diff
@@ -13,12 +13,12 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "graphql": "^15.5.1",
-    "node-sass": "^6.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-icons": "^4.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
+    "sass": "^1.62.0",
```
Comment (nicorithner) [inline package.json:21]: node-sass has been deprecated

File: src/styles/i-settings/_colors.scss (hunk 1 above)
Comment (Aferencz1987) [reply src/styles/i-settings/_colors.scss]: According to this doc, hsla is an alias for hsl so we can remove the a for the color values https://developer.mozilla.org/en-US/docs/Web/CSS/color_value

File: src/styles/i-settings/_colors.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+$color-dark: hsla(0, 0%, 0%, 1);
 
 /* === Palette === */
 $dark-purple: rgba(91, 20, 171, 1);
```
Comment (Aferencz1987) [inline src/styles/i-settings/_colors.scss]: We should make all of the color syntax match so let's change this rgba to hsl

**Review by Aferencz1987 — CHANGES_REQUESTED** [review]

File: src/styles/i-settings/_colors.scss (hunk 3 above)
Comment (nicorithner) [reply src/styles/i-settings/_colors.scss]: Good catch!
✅

**Review by Aferencz1987 — APPROVED** [review]

**Pull request merged**